pymongo/pyopenssl_context.py

```diff
@@ -37,10 +36,6 @@
 from pymongo.socket_checker import _errno_from_exception
 from pymongo.write_concern import validate_boolean
 
-_x509 = lazy_import("cryptography.x509")
-_service_identity = lazy_import("service_identity")
-_service_identity_pyopenssl = lazy_import("service_identity.pyopenssl")
```

Comment:
Does this new version work correctly when pyopenssl is installed but service_identity or cryptography is not? What should happen in that case is the driver falls back to using stdlib ssl but I suspect this new code will attempt to use pyopenssl but fail with an import error at connection establishment time.

Reply:
The most straightforward way to fix this in my eyes would be to keep these cryptograph+service_identity imports at the top-level.

Reply:
Done

Reply:
Still need to do the same with the service_identity imports.

Reply:
Done